canopen_chain_node/CMakeLists.txt

```diff
@@ -106,19 +106,25 @@ include_directories(
 )
 
 ## Declare a cpp library
-# add_library(canopen_chain_node
-#   src/${PROJECT_NAME}/canopen_chain_node.cpp
-# )
+add_library(canopen_ros_chain
+  src/ros_chain.cpp
+)
+target_link_libraries(canopen_ros_chain
+  ${Boost_LIBRARIES}
+  ${catkin_LIBRARIES}
+)
+
+## Add cmake target dependencies of the executable/library
+## as an example, message headers may need to be generated before nodes
+add_dependencies(canopen_ros_chain cob_srvs_gencpp) # TODO: workaround needed?
```

Comment:
This one slipped through: can you please change this to `add_dependencies(canopen_ros_chain ${catkin_EXPORTED_TARGETS})`